runtime/doc/gui.txt

```diff
@@ -73,25 +73,23 @@ Restart Nvim
 :restart [+cmd] [command]
                 Restarts Nvim.
 
-                1. Stops Nvim using `:qall!` (or |+cmd|, if given).
+                1. Stops Nvim using `:qall` (or |+cmd|, if given).
                 2. Starts a new Nvim server using the same |v:argv|,
                    optionally running [command] at startup. |-c|
                 3. Attaches the current UI to the new Nvim server. Other UIs
                    (if any) will not reattach on restart (this may change in
                    the future).
 
-                Use with `:confirm` to prompt if changes have been made.
-
-                Example: stop with `:qall!`, then restart: >
+                Example: discard changes and stop with `:qall!`, then restart: >
```

Comment:
```suggestion
                Example: discard changes and stop with `:qall`, then restart: >
```

Reply:
This is intentional. This used to be documentation that `:restart` was equal to `:restart +qall!` which was not communicated clearly and was already explained in other parts of the documentation. When changing `+qall!` to `+qall`, instead of keeping this example, I thought it would be better to add a new example showing how you can use `:restart +qall!` to discard changes and restart. It just happens to resemble the old thing in the diff.

Reply:
Ohh I see, thank you for explaining!